cypress/e2e/ModelizerPage/AllView/settings/git.settings.feature

```diff
@@ -0,0 +1,151 @@
+Feature: Test git configuration dialog
```

Comment:
redundancy of `settings` in file name (second not necessary)
AND
file name -> settings != test name -> configuration (need to make a choice)

Reply:
done

Reply:
second point still present

Reply:
done